code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -310,6 +310,10 @@
 			flavor_text = module_flavour
 		else
 			flavor_text = client.prefs.flavour_texts_robot["Default"]
+		// and meta info - VORE EDIT

```

Comment:
I'd like it to say specifically "Vorestation Edit" so we can easily Ctrl+F it if we need to